Sources/ApolloCodegenLib/FileGenerators/SchemaModuleFileGenerator.swift

```diff
@@ -10,10 +10,10 @@ struct SchemaModuleFileGenerator {
     _ config: ApolloCodegenConfiguration.SchemaTypesFileOutput,
     fileManager: FileManager = FileManager.default
   ) throws {
-    switch config.dependencyAutomation {
-    case let .swiftPackageManager(moduleName):
+    switch config.moduleType {
```

Comment:
Right now, if you chose other or none for `moduleType`, it's going to throw an error. If we are only supporting SPM, we need this to no-op on those `moduleType` values, not throw an error.